### ISSUE TITLE
Unable to locate package dotnet-sdk-5.0

### ISSUE THREAD

**Issue by wsbky** [dotnet/core#6612]
### Description

I'm trying to install `dotnet-sdk-5.0` in my Ubuntu 20.04 docker container. Following [this document (installing dotnet core)](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu) , I get an error `Unable to locate package dotnet-sdk-5.0`. I tried a manual install in [apt troubleshooting](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#apt-troubleshooting) but I got the same error.

I also tried some solutions in #4360: 
* https://github.com/dotnet/core/issues/4360#issuecomment-627792352
* https://github.com/dotnet/core/issues/4360#issuecomment-644446410

, but they didn't solve my problem.

### Configuration

* Host OS: macOS BigSur Version 11.5.2 (ARM64)
* Container: Ubuntu20.04

Dockerfile:
```Dockerfile
FROM ubuntu:20.04
RUN apt update && apt install -y build-essential zsh vim curl wget locales-all
RUN usermod -s /bin/zsh root
```

**Comment by mairaw** [comment]
@rbhanda @leecow can you help with this?

**Comment by leecow** [comment]
I was able to successfully install and run the latest 5.0 SDK from the 20.04 distro. Can you confirm through a manual Docker run and search for the sdk? Something like the following.

In the Mac terminal run `docker run -it ubuntu:20.04 bash` then,  at the resulting Docker bash prompt -

```
apt-get update
apt install -y wget
wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
dpkg -i packages-microsoft-prod.deb
apt-get update
apt search dotnet-sdk-5.0 # or, to list all available 5.0 versions use apt-cache madison dotnet-sdk-5.0
```
